pystac/version.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-__version__ = "1.14.1"
+__version__ = "1.14.1"  # x-release-please-version
```

Comment:
This is untested, I _think_ this comment should work but it might take some fiddling when we see the first release PR (which will be after we merge this one).